PgpWordList/PgpWordListData.cs

```diff
@@ -0,0 +1,271 @@
+using System.Collections.Generic;
+
+namespace Messerli.PgpWordList
+{
+    /// <summary>
+    /// Contains the word list from https://en.wikipedia.org/wiki/PGP_word_list.
```

Comment:
Link the URL with `<a href="...">Foo</a>`

Reply:
addressed https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/3d124497e428d3e66501bbe1ae58f200471abf31